DoLahTest/CommandUnitTest.cpp

```diff
@@ -89,6 +91,33 @@ namespace DoLahTest
             Assert::AreEqual((size_t) 5, resultVector.size());
         }
 
+        TEST_METHOD(SearchDateTaskCommand) {
+            //Arrange
+            DoLah::Calendar testCal;
+            DoLah::AbstractCommand* command;
+            command = DoLah::CommandParser::parse("add task1 by next week");
+            command->setCalendar(&testCal);
+            command->execute();
+            command = DoLah::CommandParser::parse("add task2 in 2 weeks");
+            command->setCalendar(&testCal);
+            command->execute();
+            command = DoLah::CommandParser::parse("add task3 from today to next week");
+            command->setCalendar(&testCal);
+            command->execute();
+            command = DoLah::CommandParser::parse("done 3");
+            command->setCalendar(&testCal);
+            command->execute();
+            command = DoLah::CommandParser::parse("search in 10 days");
+            command->setCalendar(&testCal);
+            command->execute();
+
+            //Act
+            std::vector<DoLah::AbstractTask*> resultVector = testCal.getSearchedTaskList();
+
+            //Assert
+            Assert::AreEqual((size_t)2, resultVector.size());
```

Comment:
Correct me if I'm wrong, the expected search result is

* task1 by next week
* task3 from today to next week